<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QLogCat</class>
 <widget class="QWidget" name="QLogCat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QLogCat</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Поиск:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leSearch">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="toolTip">
        <string>Введите строку для поиска</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbSearchLevel">
       <property name="toolTip">
        <string>Минимальный уровень</string>
       </property>
       <item>
        <property name="text">
         <string>Verbose</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Debug</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Info</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Warning</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Error</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Assert</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fatal Error</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbQt3">
       <property name="toolTip">
        <string comment="Включите эту галочку, если планируете принимать сообщения из программы, основанной на Qt3.3">Получение сообщений из Qt3.3</string>
       </property>
       <property name="text">
        <string>Qt3.3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSave">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="pic.qrc">
         <normaloff>:/pic/pic/save.png</normaloff>:/pic/pic/save.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClear">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Очистить</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="pic.qrc">
         <normaloff>:/pic/pic/clear.png</normaloff>:/pic/pic/clear.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSettings">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Настройки</string>
       </property>
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblLog">
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::DashLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>!</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Время</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Журнал</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Файл</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Строка</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Сообщение</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="gbMessage">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Сообщение</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTextEdit" name="txtMessage">
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="gbFilter">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>140</height>
          </size>
         </property>
         <property name="title">
          <string>Фильтр</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QListWidget" name="listFilter">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>80</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbAddFilter">
              <property name="toolTip">
               <string>Новый фильтр</string>
              </property>
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pbDelFilter">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="toolTip">
               <string>Удалить выделенный фильтр</string>
              </property>
              <property name="text">
               <string>Удалить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Монитор</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTableWidget" name="tblMonitor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <column>
           <property name="text">
            <string>Название</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Значение</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pbTest">
     <property name="toolTip">
      <string>Проверка</string>
     </property>
     <property name="text">
      <string>Test localhost send</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
